source/javascripts/app/version-select.js.erb

```diff
@@ -14,8 +14,8 @@ window.GUIDE_VERSIONS.onReady(function (versions) {
   });
 
   var npmInstallVersion = $('pre:contains("install -g ember-cli")');
-  //only append the version if there's one specified and not one already in the content
-  if (currentRevision.slice(0,1) !== "C" && npmInstallVersion.text().indexOf("@") === -1) {
```

Comment:
@acorncom should we leave this code so that when we want to force a version we just update the markdown to have the `@version`?